### ISSUE TITLE
Model Structure Differences: pretrained Resnet-50 torch model has different structures to the descriptions in the Resnet paper

### ISSUE THREAD

**Issue by XuefeiW** [facebook/fb.resnet.torch#117]
When I load pre-trained resnet-50 in Torch, I notice there are some differences between this model and the network scope here: http://ethereon.github.io/netscope/#/gist/db945b393d40bfa26006

Firstly, there is no Scale layer in this pre-trained network. I am not sure whether the scale layer function is achieved by other layers(eg. Batch Normalization?).

Secondly, there is a nn.SpatialBatchNormalization layer after each nn.CAddTable layer, but based on netscope(http://ethereon.github.io/netscope/#/gist/db945b393d40bfa26006) and the facebook blog(http://torch.ch/blog/2016/02/04/resnets.html), the BN layer is suppose to be added before the addition of the two branches. 
![image](https://cloud.githubusercontent.com/assets/15201418/19514026/fe1dcc82-95e9-11e6-9ea9-96b0c7dcb76f.png)
![image](https://cloud.githubusercontent.com/assets/15201418/19514084/340429fe-95ea-11e6-804e-ea026626010a.png)
![image](https://cloud.githubusercontent.com/assets/15201418/19514135/6ec83b52-95ea-11e6-9b74-f9d6b1aa4e20.png)

**Comment by XuefeiW** [comment]
Sorry, my bad.. In Torch, Scale layer is implemented in Batch Normalization layer.  Each BN layer has weight(scale factor), bias(scale bias), running_mean and running variance.
Also, someone asked about the BN positions before: https://github.com/facebook/fb.resnet.torch/issues/49
